### PR TITLE
Use new Notifications settings components using the new UI/UX everywhere 

### PR DESCRIPTION
**Pull request by jgiovaresco**
## Issue

https://gravitee.atlassian.net/browse/APIM-1426

## Description

Use the same UI/UX to configuration notifications for: V2 API, Application, Environments

V4 API
image

V2 API
image

Application
image

Environment
image

## Additional context

🏗️ Your changes can be tested here and will be available soon:
      Console: [https://pr.team-apim.gravitee.dev/6663/console](https://pr.team-apim.gravitee.dev/6663/console)
      Portal: [https://pr.team-apim.gravitee.dev/6663/portal](https://pr.team-apim.gravitee.dev/6663/portal)
      Management-api: [https://pr.team-apim.gravitee.dev/6663/api/management](https://pr.team-apim.gravitee.dev/6663/api/management)
      Gateway v4: [https://pr.team-apim.gravitee.dev/6663](https://pr.team-apim.gravitee.dev/6663)
      Gateway v3: [https://pr.gateway-v3.team-apim.gravitee.dev/6663](https://pr.gateway-v3.team-apim.gravitee.dev/6663)

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-lkomnohxsw.chromatic.com)

### CODE REVIEW

File: gravitee-apim-console-webui/src/management/application/details/notification/application-notification.component.scss [hunk 1]
```diff
@@ -0,0 +1,31 @@
+@use 'sass:map';
+@use '@angular/material' as mat;
+@use '@gravitee/ui-particles-angular' as gio;
+@use '../../../../scss/gio-layout' as gio-layout;
+
+$typography: map.get(gio.$mat-theme, typography);
```
Comment (Okhelifi) [inline gravitee-apim-console-webui/src/management/application/details/notification/application-notification.component.scss]: we can remove this line I think

File: gravitee-apim-console-webui/src/management/settings/notification/environment-notification.component.scss [hunk 2]
```diff
@@ -0,0 +1,31 @@
+@use 'sass:map';
+@use '@angular/material' as mat;
+@use '@gravitee/ui-particles-angular' as gio;
+@use '../../../scss/gio-layout' as gio-layout;
+
+$typography: map.get(gio.$mat-theme, typography);
```
Comment (Okhelifi) [inline gravitee-apim-console-webui/src/management/settings/notification/environment-notification.component.scss]: same here we can remove it I think

**Review by Okhelifi — APPROVED** [review]

File: gravitee-apim-console-webui/src/management/application/details/notification/application-notification.module.ts [hunk 3]
```diff
@@ -14,30 +14,20 @@
...
-    MatButtonModule,
-    MatFormFieldModule,
-    MatInputModule,
-    MatOptionModule,
-    MatSelectModule,
-  ],
+@NgModule({
+  declarations: [ApplicationNotificationComponent],
+  exports: [],
+  imports: [CommonModule, MatCardModule, MatButtonModule, MatIconModule, MatDialogModule, NotificationModule, MatLegacySnackBarModule],
```
Comment (ThibaudAV) [inline gravitee-apim-console-webui/src/management/application/details/notification/application-notification.module.ts]: we don't need MatLegacySnackBarModule anymore. i think you can juste remove it here

File: gravitee-apim-console-webui/src/management/settings/notification/environment-notification.module.ts [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+import { MatLegacySnackBarModule } from '@angular/material/legacy-snack-bar';
+
+import { EnvironmentNotificationComponent } from './environment-notification.component';
+
+import { NotificationModule } from '../../../components/notification';
+
+@NgModule({
+  declarations: [EnvironmentNotificationComponent],
+  exports: [],
+  imports: [CommonModule, MatCardModule, MatButtonModule, MatIconModule, MatDialogModule, NotificationModule, MatLegacySnackBarModule],
```
Comment (ThibaudAV) [inline gravitee-apim-console-webui/src/management/settings/notification/environment-notification.module.ts]: We don't need MatLegacySnackBarModule anymore. i think you can juste remove it here

**Review by ThibaudAV — APPROVED** [review]

**Pull request merged**